vitoriapinheiro/ChallengeImPada
# implemented braille dots

# What I did

- Presented braille dots and componentized
- Add the code for all the algarisms


# How to test

1. Clone this repository (`git clone https://github.com/vitoriapinheiro/ChallengeImPada.git`)
2. Open the project on Xcode
3. Run the app on a IPhone device 
4. Open the app

- [x] Check if the vibration is working while you press the red/blue dots
- [x] Check if you can only use the app on lanscape mode
- [x] Check if the code is well componentized

## How long I took
1 day
